crates/bevy_reflect/src/type_path.rs

```diff
@@ -0,0 +1,26 @@
+pub trait TypePath: 'static {
```

Comment:
Needs top level doc strings explaining why it exists, what it's used for, and why not just use `TypeId`.